flow/project.py

```diff
@@ -681,24 +694,14 @@ class FlowGroupEntry:
         The :meth:`FlowProject.run` options to pass when submitting the group.
         These will be included in all submissions. Submissions use run
         commands to execute.
-
+    aggregator : :class:`~.aggregator`
+        aggregator object associated with the :class:`FlowGroup` (Default value = None).
     """
 
-    # TODO: Enable aggregator argument in documentation.
-    # aggregator : :class:`~._aggregator`
-    #     aggregator object associated with the :class:`FlowGroup`. If None, the
-    #     default aggregator is used (Default value = None).
-
-    def __init__(self, name, options=""):
-        # TODO: Enable aggregator argument.
-        # def __init__(self, name, options="", aggregator=None):
+    def __init__(self, name, options="", aggregator=None):
         self.name = name
         self.options = options
-        self.aggregator = _aggregator.groupsof(1)
-        # TODO: Enable aggregator argument.
-        # if aggregator is None:
-        #     aggregator = _aggregator.groupsof(1)
-        # self.aggregator = aggregator
+        self.aggregator = aggregator
```

Comment:
Discussed with @kidrahahjo: This has to do with delayed initialization. We cannot know at this point whether there is an aggregator affiliated with the operation. We added this comment at another point in the code, and I copied it here in 1680f46f2f5755e053efe0ab243c1f9b045575a7.
```
        # We register aggregators associated with operation functions in
        # `_register_groups` and we do not set the aggregator explicitly.
        # We delay setting the aggregator because we do not restrict the
        # decorator placement in terms of `@FlowGroupEntry`, `@aggregator`, or
        # `@operation`.
```